MonoGame.Framework/Game.cs

```diff
@@ -19,6 +19,10 @@ namespace Microsoft.Xna.Framework
 {
     public class Game : IDisposable
     {
+#if DESKTOPGL
+        public static bool Resizable = false;

```

Comment:
SDL does not have a way of setting window resizable parameter after the window has already been created.